src/model/class.ts

```diff
@@ -151,7 +151,7 @@ export class Class implements Feature {
 
     this.warnings =
         init.warnings === undefined ? [] : Array.from(init.warnings);
-    this.demos = init.demos || jsdocLib.extractDemos(init.jsdoc, document.url);
+    this.demos = init.demos || jsdocLib.extractDemos(init.jsdoc);
```

Comment:
So this is happening in `Class` so this is at resolution time. The prototype chain of `Scanned*` is kinda crazy, but as of fairly recently, it's very sane for resolved features. I think this is the only place this is needed, so long as we don't care that scanned features don't have demos, which seems fine to me.